roles/ceph-common/meta/main.yml

```diff
@@ -8,3 +8,4 @@ dependencies:
     repos:
       - repo: 'deb {{ apt_repos.ceph.repo }}{{ ceph.stable_release }}/ {{ ansible_lsb.codename }} main'
         key_url: '{{ apt_repos.ceph.key_url }}'
+    when: ansible_architecture != "ppc64le"
```

Comment:
Shouldn't the conditional go to the ceph roles in general, rather than the part of the ceph role that would add a repository?